src/main/java/com/github/technus/tectech/nei/IMCForNEI.java

```diff
@@ -0,0 +1,46 @@
+package com.github.technus.tectech.nei;
+
+import cpw.mods.fml.common.Loader;
+import cpw.mods.fml.common.event.FMLInterModComms;
+import net.minecraft.nbt.NBTTagCompound;
+
+public class IMCForNEI {
+
+    public static void IMCSender() {
+        if (!Loader.isModLoaded("NotEnoughItems")) {
+            return;
+        }
+        sendHandler("gt.recipe.eyeofharmony", "gregtech:gt.blockmachines:15410", 1);
+        sendCatalyst("gt.recipe.eyeofharmony", "gregtech:gt.blockmachines:15410");
+    }
+
+    private static void sendHandler(String aName, String aBlock, int aMaxRecipesPerPage) {
+        NBTTagCompound aNBT = new NBTTagCompound();
+        aNBT.setString("handler", aName);
+        aNBT.setString("modName", "GregTech");
```

Comment:
Why not TecTech? (If you switch use a ref to mod id and name instead of string btw)